lib/edge_commander_web/controllers/sims_controller.ex

```diff
@@ -1,13 +1,15 @@
 defmodule EdgeCommanderWeb.SimsController do
   use EdgeCommanderWeb, :controller
-  import EdgeCommander.ThreeScraper, only: [all_sim_numbers: 0, get_last_two_days: 1, get_all_records_for_sim: 1, get_single_sim: 1]
-  import EdgeCommander.Nexmo, only: [get_message: 1, get_single_sim_messages: 1]
+  import EdgeCommander.ThreeScraper, only: [get_sim_numbers: 1, get_last_two_days: 1, get_all_records_for_sim: 1, get_single_sim: 1, get_single_sim_by_user: 2, get_all_records_for_sim_by_user: 2, get_all_users_by_number: 1]
```

Comment:
Remove only part, It supposed to be used only when you are importing a few methods, not a FULL module. So just import `import EdgeCommander.ThreeScraper` and remove else part.